### DISCUSSION TITLE
Is there a keybinding for maximizing the bottom dock?

### DISCUSSION THREAD

**Discussion by nixpulvis** [zed-industries/zed#8066] [Feature Requests]
I was searching through https://zed.dev/docs/key-bindings for a way to maximize the bottom dock, but I couldn't find it.

If it's not already possible, I would love to add it if someone could kindly point me towards where to start.

Loving Zed so far. Truly reminds me of happier days back with Atom.

**Comment by Moshyfawn** [comment]
`Shift + Esc` to toggle the zoom/focus of a pane.

**Comment by nixpulvis** [comment]
I might suggest you change the name "Toggle zoom" to something that doesn't imply scaling but thank you for pointing me to the setting.
